src/components/Button/index.tsx

```diff
@@ -227,19 +227,20 @@ export const ButtonEmpty = styled(BaseButton)`
   }
 `
 
-export const ButtonText = styled(BaseButton)`
+export const ButtonText = styled(BaseButton)<{ color?: string; noUnderline?: boolean }>`
```

Comment:
can the `SwapBuyFiatButton` component just inherit from this and then override color + the focus/active states there? i prefer that over trying to finesse these shared components into more generalized versions 😬

Reply:
yeppp that is wayyy cleaner, ty done.